### PR TITLE
Bugfix/156692067 - Blank experiment species

### PR DESCRIPTION
**Pull request by monicajianu**
- Added species retrival from organism characteristic column in condensed sdrf file
- Refactored CondensedSdrf parser to remove dependency on IdfParser
- Refactored IdfParser and added retrieval of expected clusters, publication titles and DOIs
- Implemented testing utility method that allows retrieval of all experiment accessions available in the database, so that exhaustive testing can be performed

### CODE REVIEW

File: base/src/main/java/uk/ac/ebi/atlas/experimentimport/idf/IdfParser.java [hunk 1]
```diff
@@ -35,16 +39,11 @@ public IdfParser(IdfStreamerFactory idfReaderFactory) {
...
                             .filter(line -> line.length > 1)
-                            .filter(line -> INVESTIGATION_TITLE_ID.equalsIgnoreCase(line[0].trim()) ||
-                                        AE_EXPERIMENT_DISPLAY_NAME_ID.equalsIgnoreCase(line[0].trim()) ||
-                                        PUBMED_ID.equalsIgnoreCase(line[0].trim()) ||
-                                        PUBLICATION_TITLE_ID.equalsIgnoreCase(line[0].trim()) ||
-                                        EXPECTED_CLUSTERS_ID.equalsIgnoreCase(line[0].trim()))
+                            .filter(line -> LINE_IDS.contains(line[0].trim().toUpperCase()))
                             .collect(Collectors.toMap(
-                                    line -> line[0].toUpperCase().trim(),
+                                    line -> line[0].trim().toUpperCase(),
```
Comment (alfonsomunozpomer) [inline base/src/main/java/uk/ac/ebi/atlas/experimentimport/idf/IdfParser.java]: Maybe you can `.map(line -> line[0].toUpperCase().trim())` and then filter/collect?

**Comment by alfonsomunozpomer** [comment]
There was a version mismatch with Surefire/Failsafe in the parent POM and the modules POM, so rebase this branch on dev and give it a go to get a green build.

File: base/src/main/java/uk/ac/ebi/atlas/experimentimport/idf/IdfParser.java (hunk 1 above)
Comment (monicajianu) [reply base/src/main/java/uk/ac/ebi/atlas/experimentimport/idf/IdfParser.java]: I'm not sure what you mean. I am building a map with `line[0]` as key and `line[1:]` as value. If I map the whole thing to `line[0].toUpperCase().trim()` then I'll lose the rest of the line contents...

**Comment by monicajianu** [comment]
Green build: http://bar.ebi.ac.uk:8085/browse/ATLAS-SCXACIDEVTASKS8-4

**Pull request merged**